app/controllers/foreman_leapp/api/v2/report_controller.rb

```diff
@@ -0,0 +1,25 @@
+module ForemanLeapp
+  module Api
+    module V2
+      class ReportController < ::Api::V2::BaseController
+  
+        def create
+          require 'date'
```

Comment:
This shouldn't be necessary. If it is necessary, then requires should be done outside of methods so they're performed once at load time.

Reply:
Thanks, not needed indeed